stetho-timber/src/main/java/com/facebook/stetho/timber/StethoTree.java

```diff
@@ -54,10 +75,63 @@ protected void log(int priority, String tag, String message, Throwable t) {
         logLevel = Console.MessageLevel.LOG;
     }
 
+    StringBuilder messageBuilder = new StringBuilder();
+
+    if(mConfiguration.mShowTags && tag != null) {
+      messageBuilder
+          .append("[")
+          .append(tag)
+          .append("] ");
+    }
+
+    messageBuilder.append(message);
+
     CLog.writeToConsole(
         logLevel,
         Console.MessageSource.OTHER,
-        message
+        messageBuilder.toString()
     );
   }
-}
+
+  public static class Configuration {
```

Comment:
nit: this appears to be two classes where 1 would do:

  StethoTree tree = new StethoTree.Builder()
    .setStuff()
    .build();

Since the configuration doesn't need to be exposed as its own separate class it can just be merged with StethoTree.